paper_io_sim.tex

```diff
@@ -837,14 +854,16 @@
             \end{figure*}
         \subsection{Concurrent applications (Exp 2)}
 
-            The page cache model notably reduced \wrench's simulation error
+            \todo[inline]{HC: I don't really see a plateau in Figure~\ref{fig:multi_local}}
```

Comment:
The plateau is between 1 and 7 concurrent executions in Fig 5-right (writes). I clarified the text.